system-parachains/bridge-hubs/common/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "bridge-hub-common"
```

Comment:
I suggest to keep using existing `bp_bridge_hub_cumulus` unless there are any problems with it. The `bp_bridge_hub_cumulus` is also used by Rococo/Westend BH primitives, so I do not see any point in having two copies of this crate. For other crates it is ok - we'll solve that in https://github.com/paritytech/parity-bridges-common/issues/2680 during migration to `polkadot-sdk` - either well have several relay versions (one for P<>K here, in fellowship and another one for R<>W in polkadot-sdk) or we will revert to a single BH primitives crate in polkadot-sdk

Reply:
@bkontur Slava, sure, no problem, do as you think is the best (I know you know :) ), revert/remove whatever, no problem, merge when you ok with that. I have a feeling that cumulus word will be obsolete soon :)